submissions/morenko/memory-game/js/index.js

```diff
@@ -0,0 +1,182 @@
+// создадим массив с картами
+const listCards = [
+    {
+        name: 1,
+        front: 'assets/front/1.jpg'
+    },
+    {
+        name: 2,
+        front: 'assets/front/2.jpg'
+    },
+    {
+        name: 3,
+        front: 'assets/front/3.jpg'
+    },
+    {
+        name: 4,
+        front: 'assets/front/4.jpg'
+    },
+    {
+        name: 5,
+        front: 'assets/front/5.jpg'
+    },
+    {
+        name: 6,
+        front: 'assets/front/6.jpg'
+    },
+    {
+        name: 7,
+        front: 'assets/front/7.jpg'
+    },
+    {
+        name: 8,
+        front: 'assets/front/8.jpg'
+    },
+];
+//console.log(cards);
+
+// создадим копию массива
+const copylistCards = listCards.slice();
+//console.log(copyCards);
+
+const cards = listCards.concat(copylistCards); //объединим два одинаковых массива карт
+//console.log(cards); //проверим
+let countCards = cards.length; // длина массива - количество карт в нем
+
+// функция сортировки меняет порядок элементов массива случайным образом
+const shuffle = () => cards.sort(() => Math.random() - 0.5);
+//console.log(shuffle());
+
+// найдем кнопку по классу, повесим на нее кликер
+const reset = document.querySelector(".reset");
+
+// при клике на кнопку запустим функцию перезагрузки карт в дом
+reset.addEventListener('click', function () {
+    //console.log(shuffle());
+    reloadBoard();
+});
+
+// создадим функцию заполняющую доску картами
+// найдем в доме доску по классу
+const board = document.querySelector('.board'); // доска для карт
+
+const createCards = function() {
+    let arrCards = ''; //создадим пустой массив дом-елементов
+    cards.forEach(({name, front}) => {
```

Comment:
It will be better if you will pass `cards` as an argument into function and not use global variables.

Rule of thumb - global variables are bad and you should avoid them as much as possible.